platform/src/main/java/platform/pageobjects/Header/Header.java

```diff
@@ -0,0 +1,390 @@
+		log.info("Opened Submenu");
+		CommonFunctions.isElementVisible(educationCostConsultationSupportLink);
+		log.info("Submenu : Education Cost Consultation Support Link is visible");
+
+	}
+
+	// Method : Checks the Header menu links are working by clicking on them
+	public void checkHeaderLinksWorking() throws Exception {
+
+		CommonFunctions.isElementVisible(examinationWindowLink);
+		CommonFunctions.waitForVisiblity(examinationWindowLink, waitTime);
+		examinationWindowLink.click();
+		log.info("Clicked on Examination Window page");
+
+		CommonFunctions.isElementVisible(learningWindowLink);
+		CommonFunctions.waitForVisiblity(learningWindowLink, waitTime);
+		learningWindowLink.click();
+		log.info("Clicked on Learning Window page");
+
+		CommonFunctions.isElementVisible(informationWindowLink);
+		CommonFunctions.waitForVisiblity(informationWindowLink, waitTime);
+		informationWindowLink.click();
+		log.info("Clicked on Information Window page");
+
+		CommonFunctions.isElementVisible(productListLink);
+		CommonFunctions.waitForVisiblity(productListLink, waitTime);
+		productListLink.click();
+		log.info("Clicked on Product List page");
+
+		CommonFunctions.isElementVisible(signUpLink);
+		CommonFunctions.waitForVisiblity(signUpLink, waitTime);
+		signUpLink.click();
+		log.info("Clicked on SignUp page");
+		CommonFunctions.isElementVisible(emailTextBox);
+		log.info("Email Textbox is visible");
+		CommonFunctions.isElementVisible(EIDTextbox);
+		log.info("EID Textbox is visible");
+		CommonFunctions.isElementVisible(sendConfirmationEmailButton);
+		log.info("Send confirmation Button is visible");
+
+		driver.navigate().back();
+		log.info("Back on Previous Page");
+
+		CommonFunctions.isElementVisible(logo);
+		CommonFunctions.waitForVisiblity(logo, waitTime);
+		logo.click();
+		log.info("Back on Home Page");
+
+		CommonFunctions.isElementVisible(loginLink);
+		CommonFunctions.waitForVisiblity(loginLink, waitTime);
+		loginLink.click();
+		log.info("Clicked on Login Page");
+		CommonFunctions.isElementVisible(emailTextBox);
+		log.info("Email Textbox is visible");
+		CommonFunctions.isElementVisible(EIDTextbox);
+		log.info("EID Textbox is visible");
+		CommonFunctions.isElementVisible(loginButton);
+		log.info("Login Button is visible");
+
+		driver.navigate().back();
+		log.info("Back on Previous Page");
+
+	}
+
+	// Method : Checks the Header SubMenu links are working by clicking on them
+	public void checkSubMenuLinkWorking() throws Exception {
+
+		Actions actions = new Actions(driver);
+
+		CommonFunctions.waitandClear(examinationWindowLink, waitTime);
+		actions.moveToElement(examinationWindowLink).perform();
+		log.info("Opened Submenu");
+		CommonFunctions.isElementVisible(testTopLink);
+		CommonFunctions.waitForVisiblity(testTopLink, waitTime);
+		testTopLink.click();
+		log.info("Opened Test Top submenu");
+		driver.navigate().back();
+		log.info("Back on Previous Page");
+
```

Comment:
It seems you are performing same steps in the code from 269-278
Plz write a method and pass links as parameters

Reply:
@csonparote 
Can you please elaborate ?

Reply:
U can also pass an array of key value pair to clickSubMenu() to verify links .
NVM this is good too.
No changes needed.
Resolving conversation.